modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/service/impl/DDMFieldLocalServiceImpl.java

```diff
@@ -533,10 +538,15 @@ private void _collectDDMFieldInfos(
 					String languageId = LanguageUtil.getLanguageId(
 						entry.getKey());
 
+					String sanitizedValue = SanitizerUtil.sanitize(
+						0, 0, 0, DDMFieldAttribute.class.getName(), 0,
+						ContentTypes.TEXT_HTML,
+						_sanitize(ddmFormField.getType(), entry.getValue()));
+
```

Comment:
@rodrigopaulino Why do we need to call ` SanitizerUtil.sanitize` twice? One here and one in `private String _sanitize(String ddmFormFieldType, String value)`

Reply:
We don't! I'll fix this right away. Thanks, @carolmariaabb

Reply:
Done, @carolmariaabb